EU4toV2/Source/EU4World/Modifiers/Modifiers.cpp

```diff
@@ -26,37 +22,26 @@ void EU4::Modifiers::initialize()
 void EU4::Modifiers::processFolder(const std::string& folderName)
 {
 	if (commonItems::DoesFolderExist(theConfiguration.getEU4Path() + "/common/" + folderName))
-	{
-		auto filenames = commonItems::GetAllFilesInFolder(theConfiguration.getEU4Path() + "/common/" + folderName + "/");
-		for (const auto& filename : filenames)
-		{
+		for (const auto& filename: commonItems::GetAllFilesInFolder(theConfiguration.getEU4Path() + "/common/" + folderName + "/"))
 			parseFile(theConfiguration.getEU4Path() + "/common/" + folderName + "/" + filename);
-		}
-	}
-	for (const auto& modName : theConfiguration.getEU4Mods())
-	{
-		if (commonItems::DoesFolderExist(modName + "/common/" + folderName))
-		{
-			auto filenames = commonItems::GetAllFilesInFolder(modName + "/common/" + folderName + "/");
-			for (const auto& filename : filenames)
-			{
-				parseFile(modName + "/common/" + folderName + "/" + filename);
-			}
-		}
-	}
+
+	for (const auto& modName: theConfiguration.getEU4Mods())
+		for (const auto& filename: commonItems::GetAllFilesInFolder(modName + "/common/" + folderName + "/"))
+			parseFile(modName + "/common/" + folderName + "/" + filename);
 }
 
 void EU4::Modifiers::registerKeys()
 {
-	registerRegex("[a-zA-Z0-9_]+", [this](const std::string& modifierName, std::istream& theStream) {
+	registerRegex(commonItems::catchallRegex, [this](const std::string& modifierName, std::istream& theStream) {
 		Modifier modifier(theStream);
 		modifiers.insert(std::make_pair(modifierName, modifier));
 	});
 }
 
 std::optional<EU4::Modifier> EU4::Modifiers::getModifier(const std::string& modifierName) const
 {
-	if (modifiers.count(modifierName) > 0) return std::make_optional(modifiers.at(modifierName));
-
-	return std::nullopt;
+	if (modifiers.count(modifierName))
+		return modifiers.at(modifierName);
+	else
```

Comment:
The else is not needed here, but I guess you intentionally added it to make code clearer.

Reply:
Yup.